### PR TITLE
perf(crypto): apply `BatchMapScalarFieldToPoint()`

### PR DESCRIPTION
**Pull request by chokobole**
# Description

This PR optimizes kzg setup by applying `BatchMapScalarFieldToPoint()`

### CODE REVIEW

File: tachyon/crypto/commitments/kzg/kzg.h [hunk 1]
```diff
@@ -61,13 +61,9 @@ class KZG {
...
-    if (!G1Point::MultiScalarMul(powers_of_tau, g1,
-                                 &g1_powers_of_tau_jacobian)) {
-      return false;
-    }
     g1_powers_of_tau_.resize(size);
-    if (!math::ConvertPoints(g1_powers_of_tau_jacobian, &g1_powers_of_tau_)) {
+    if (!G1Point::BatchMapScalarFieldToPoint(g1, powers_of_tau,
+                                             &g1_powers_of_tau_)) {
       return false;
     }
```
Comment (TomTaehoonKim) [inline tachyon/crypto/commitments/kzg/kzg.h:68]:
```suggestion
    if (!G1Point::BatchMapScalarFieldToPoint(g1, powers_of_tau, &g1_powers_of_tau_)) return false;
```
I'm not sure we have unified code in our repo like above suggestion, but I think I've seen this style more frequently. How about fixing like this?

Comment (chokobole) [reply tachyon/crypto/commitments/kzg/kzg.h:68]: IMHO, it's up to developers for the things not described in google style if this doesn't impact on performance!

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

**Review by dongchangYoo — APPROVED** [review]
LGTM

**Review by Insun35 — APPROVED** [review]
LGTM

**Pull request merged**